extensions/ql-vscode/src/vscode-tests/minimal-workspace/activation.test.ts

```diff
@@ -1,35 +1,36 @@
-import * as assert from "assert";
 import * as path from "path";
 import * as vscode from "vscode";
 import * as determiningSelectedQueryTest from "./determining-selected-query-test";
 
-describe("launching with a minimal workspace", async () => {
+// Temporary until Mocha is fully removed. This is necessary for passing timeouts to `it`.
+declare let it: jest.It;
+
+describe("launching with a minimal workspace", () => {
   const ext = vscode.extensions.getExtension("GitHub.vscode-codeql");
   it("should install the extension", () => {
-    assert(ext);
+    expect(ext);
```

Comment:
I think this will need some condition, such as:

```suggestion
    expect(ext).not.toBeUndefined();
```